proofs/eo/cpc/programs/Utils.eo

```diff
@@ -21,6 +21,45 @@
 (declare-const @list.nil @List)
 (declare-const @list (-> (! Type :var T :implicit) T @List @List) :right-assoc-nil @list.nil)
 
+
+; program: $get_arg_list_rec
+; args:
+; - t T: The term to inspect.
```

Comment:
I think this needs to be updated, but I'm not sure. I'm at a loss of which of the parameters below count as arguments.

Reply:
The function is (-> S @List @List).  The parameters are not the same as the arguments, they are used to define the function but the argument list is different.

Reply:
I see... I was confused because a parameter had the same name of an argument, and the arguments are not named in the definition, only in the doc. This seems not ideal...